services/harmony/env-defaults

```diff
@@ -365,120 +365,104 @@ HARMONY_GDAL_ADAPTER_REQUESTS_MEMORY=128Mi
 HARMONY_GDAL_ADAPTER_LIMITS_CPU=128m
 HARMONY_GDAL_ADAPTER_LIMITS_MEMORY=8Gi
 HARMONY_GDAL_ADAPTER_INVOCATION_ARGS='python -m gdal_subsetter'
-HARMONY_GDAL_ADAPTER_SERVICE_QUEUE_URLS='["ghcr.io/nasa/harmony-gdal-adapter:latest,http://localstack:4566/queue/harmony-gdal-adapter.fifo"]'
```

Comment:
Are we now picking these environment variables up from packages/util? We might be able to update the bin/generate-new-service script too.

Reply:
yes, I think it is always the case. These env vars are not used at all.